### PR TITLE
add sse support for the accumulate function

### PR DESCRIPTION
**Pull request by 0x72D0**
the performance for the sse support:

screenshot_20181022_211751

### CODE REVIEW

File: src/image_function_simd.cpp [hunk 1]
```diff
@@ -1326,6 +1369,7 @@ if ( simdType == neon_function ) { \
         const uint32_t nonSimdWidth = width - totalSimdWidth;
 
         AVX_CODE( avx::Accumulate( rowSize, imageY, imageYEnd, outY, simdWidth, totalSimdWidth, nonSimdWidth ); )
+        SSE_CODE( sse::Accumulate( rowSize, imageY, imageYEnd, outY, simdWidth, totalSimdWidth, nonSimdWidth ); )
```
Comment (ihhub) [inline src/image_function_simd.cpp]: You forgot to change line **1348** and your code never reaches this line. That line should be `if( (simdType == cpu_function) || (simdType == neon_function) || ((width * height * colorCount) < simdSize) ) {`

**Comment by 0x72D0** [comment]
this is the cpu I use for the test, a x86_64:

screenshot_20181022_213121

**Comment by ihhub** [comment]
@0x72D0 if you facing a consistent crash with blob detection code could you please create a separate issue for this describing the problem and adding more information? Ideally if you run only blob detection in debug mode so you could define the place of crash.

**Comment by 0x72D0** [comment]
ok I'm gonna add a new issue

File: src/image_function_simd.cpp (hunk 1 above)
Comment (0x72D0) [reply src/image_function_simd.cpp]: ok it should be fix now

**Comment by ihhub** [comment]
@0x72D0, thanks again for you contribution to the project!

**Pull request merged**